pennylane/qchem/tapering.py

```diff
@@ -447,10 +470,9 @@ def taper_hf(generators, paulixops, paulix_sector, num_electrons, num_wires):
 
     fermop_terms_as_ps = [pauli_sentence(term) for term in fermop_terms]
     ferm_ps = functools.reduce(lambda i, j: i * j, fermop_terms_as_ps)
-    ferm_op = ferm_ps.hamiltonian()
 
     # taper the HF observable using the symmetries obtained from the molecular hamiltonian
-    fermop_taper = taper(ferm_op, generators, paulixops, paulix_sector)
+    fermop_taper = taper(ferm_ps, generators, paulixops, paulix_sector)
```

Comment:
If we pulled out a `_taper_pauli_sentence` private function, we could use it here and not have to convert back and forth between representations.

Reply:
Thanks @albi3ro, we have a plan for updating `qchem` with the new Pauli objects/arithmetic. Will consider this and fix everywhere as part of that work.